pkg/jsonlog/jsonlogbytes.go

```diff
@@ -38,6 +39,15 @@ func (mj *JSONLogs) MarshalJSONBuf(buf *bytes.Buffer) error {
 		buf.WriteString(`"stream":`)
 		ffjsonWriteJSONString(buf, mj.Stream)
 	}
+	if len(mj.Tag) > 0 {
+		if first {
+			first = false
+		} else {
+			buf.WriteString(`,`)
+		}
+		buf.WriteString(`"tag":`)
+		ffjsonWriteJSONString(buf, mj.Tag)
+	}
```

Comment:
Please add a case that uses `tag` to the list of testcases for this function

Reply:
Done. Added an extra log line with a tag to the existing unit test for it.